crates/dojo-project/src/migration/mod.rs

```diff
@@ -219,6 +210,19 @@ impl Deployable for ContractMigration {
         ]
         .concat();
 
+        let contract_address = get_contract_address(
```

Comment:
@kariy for some reason, it seems the computed address here doesn't match the actual deploy address. since i still get a failed txn on deploying a contract that is already deployed